Examples/Logging Tracer/LoggingTracer.swift

```diff
@@ -69,5 +69,23 @@ class LoggingTracer: Tracer {
         func setActive(_ active: Bool) -> Self {
             return self
         }
+
+        func withActiveSpan<T>(_ operation: (any SpanBase) throws -> T) rethrows -> T {
```

Comment:
should this be in an `#else` to the `#if canImport(_Concurrency)`? It seems like this my result in duplicate symbols when _Concurrency is imported?  Or does the fact it's async cause it not to conflict?